Report/CostBreakdownReporter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace MauticPlugin\MauticMediaBundle\Report;
+
+use Doctrine\Common\Cache\CacheProvider;
+use Doctrine\DBAL\Cache\QueryCacheProfile;
+use Doctrine\ORM\EntityManager;
+use Mautic\CoreBundle\Helper\Chart\ChartQuery;
+
+class CostBreakdownReporter
+{
+    /**
+     * @var EntityManager
+     */
+    private $em;
+
+    /**
+     * @var CacheProvider
+     */
+    private $cache;
+
+    /**
+     * CostBreakdownReport's constructor.
+     *
+     * @param EntityManager $em
+     * @param CacheProvider $cache
+     */
+    public function __construct($em, $cache = null)
+    {
+        $this->em    = $em;
+        $this->cache = $cache;
+    }
+
+    /**
+     * Get the report with caching if available.
+     *
+     * @param int       $campaignId The campaign's ID
+     * @param \DateTime $dateFrom
+     * @param \DateTime $dateTo
+     */
+    public function getReport($campaignId, $dateFrom, $dateTo)
+    {
+        $timeInterval     = DatePadder::getTimeUnitFromDateRange($dateFrom, $dateTo);
```

Comment:
Same here. Cant remember what happens if you dont have a Use statement, for static called stuff.

Reply:
Same as above.